.github/workflows/golangci-lint.yml

```diff
@@ -28,8 +25,9 @@ jobs:
 
           # Optional: golangci-lint command line arguments.
           # args: --issues-exit-code=0
-          args: --timeout=3m
```

Comment:
this was added on purpose IIRC - https://github.com/golangci/golangci-lint-action/issues/308